### PR TITLE
Update starterpack

### PR DESCRIPTION
**Pull request by ru-fu**
Update the starter pack to the current version, and fix the link colour for dark mode.

Fixes https://github.com/canonical/lxd/issues/13062
Fixes https://github.com/canonical/lxd/issues/12244

### CODE REVIEW

**Comment by tomponline** [comment]
@ru-fu I linked this to https://github.com/canonical/lxd/issues/13062 so it closes it when merged, is that correct?

**Comment by ru-fu** [comment]
> @ru-fu I linked this to #13062 so it closes it when merged, is that correct?

Yes - I have the link in the one commit that actually fixes this part.

**Review by tomponline — APPROVED** [review]

**Comment by tomponline** [comment]
> Yes - I have the link in the one commit that actually fixes this part.

Thanks!

File: Makefile [hunk 1]
```diff
@@ -131,6 +131,7 @@ update-metadata: build
 .PHONY: doc-setup
 doc-setup: client
 	@echo "Setting up documentation build environment"
+	pip3 install gitpython pyyaml
```
Comment (tomponline) [inline Makefile:134]: Could this be done just after line 136  so its installed in the venv and not globally?

Comment (ru-fu) [reply Makefile:134]: Hmm, not sure. It might work ... can't test it locally though because it works on my machine anyway. ;)
Let's try it.

**Pull request merged**